wix/main.wxs

```diff
@@ -60,103 +60,79 @@
-                                Type='string'
-                                Value='[INSTALLDIR]'
-                                KeyPath='yes' />
-                            <Environment
-                                Id='INSTALLPATH'
-                                Name='PATH'
-                                Value='[INSTALLBINDIR]'
-                                Permanent='no'
-                                Part='first'
-                                Action='set'
-                                System='yes' />
-                        </Component>
-                    </Directory>
                 </Directory>
             </Directory>
-            <Component Id='UserBinDir' Guid='*' Win64='$(var.Win64)'>
-                <RegistryValue
-                    Root='HKCU'
-                    Key='Software\[Manufacturer]\[ProductName]'
-                    Name='BinDir'
-                    Type='string'
-                    Value='[LocalAppDataFolder]Volta\bin'
-                    KeyPath='yes' />
-                <Environment
-                    Id='BINPATH'
-                    Name='PATH'
-                    Value='[LocalAppDataFolder]Volta\bin'
-                    Permanent='no'
-                    Part='first'
-                    Action='set'
-                    System='no' />
-            </Component>
-            <Merge Id='VCRedist' SourceFile='wix\Microsoft_VC140_CRT_x64.msm' DiskId='1' Language='0'/>
+            <Merge Id='VCRedist' SourceFile='wix\Microsoft_VC140_CRT_x64.msm' DiskId='1' Lanuage='0'/>
         </Directory>
 
-        <ComponentGroup Id='Binaries' Directory='INSTALLBINDIR'>
+        <ComponentGroup Id='Binaries' Directory='INSTALLDIR'>
             <Component Id='voltaBinary' Guid='*' Win64='$(var.Win64)'>
                 <File
                     Id='voltaEXE'
                     Name='volta.exe'
                     DiskId='1'
                     Source='target\release\volta.exe'
                     KeyPath='yes'/>
+                <Environment
+                    Id='INSTALLPATH'
+                    Name='PATH'
+                    Value='[INSTALLDIR]'
+                    Permanent='no'
+                    Part='first'
+                    Action='set'
+                    System='yes' />
+            </Component>
+            <Component Id='shimBinary' Guid='*' Win64='$(var.Win64)'>
+                <File
+                    Id='voltashimEXE'
+                    Name='volta-shim.exe'
+                    DiskId='1'
+                    Source='target\release\volta-shim.exe'
+                    KeyPath='yes'/>
+            </Component>
+            <Component Id='migrateBinary' Guid='*' Win64='$(var.Win64)'>
+                <File
+                    Id='voltamigrateEXE'
+                    Name='volta-migrate.exe'
+                    DiskId='1'
+                    Source='target\release\volta-migrate.exe'
+                    KeyPath='yes'/>
```

Comment:
How have we tested this? Just want to make sure it's solid before we ship it!

Reply:
(Note that it looks *right* to me, but I figured I'd double-check. Same goes for the \*n\*x installers.)

Reply:
I tested the installers before this in a couple of beta releases, see: https://github.com/volta-cli/volta/releases/tag/v0.7.0-beta2

I definitely agree we need to make sure they're solid, I think we can cut a 0.7.0-beta3 after merging this and do some more testing (that will also allow me to do some internal tests on distribution).